app/javascript/designSystem/__stories__/fieldComponents.stories.jsx

```diff
@@ -0,0 +1,290 @@
+import { h } from 'preact';
+import { storiesOf } from '@storybook/react';
+
+import './typography.scss';
+
+storiesOf('Base/Components', module).add('Field Component', () => (
```

Comment:
I think _Field component_ could be easily combined with _Form component_...

Reply:
Done.